apps/mobile/src/libs/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+	type Control,
+	type FieldErrors,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+import { RadioGroup as RNRadioGroup } from "react-native-radio-buttons-group";
+
+import { Text, View } from "~/libs/components/components";
+import { BaseColor } from "~/libs/enums/enums";
+import { useFormController } from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import { type RadioButtonProps, type ViewStyle } from "~/libs/types/types";
+
+import { CIRCLE_SIZE } from "./libs/constants/constants";
+import { BorderSize } from "./libs/enums/enums";
+import { styles } from "./styles";
+
+type RadioGroupItemStyle = {
+	circleBorderColorActive?: string;
+	circleBorderSize?: number;
+	circleBorderSizeActive?: number;
+	circleColorActive?: string;
+	circleSize?: number;
+} & ViewStyle;
+
+type RadioGroupItem = {
+	label: string;
+	value: string;
+};
+
+type Properties<T extends FieldValues> = {
+	control: Control<T>;
+	errors: FieldErrors<T>;
+	itemContainerStyle?: RadioGroupItemStyle;
+	name: FieldPath<T>;
+	onChange?: (value: RadioGroupItem) => void;
+	options: RadioGroupItem[];
+};
+
+const RadioGroup = <T extends FieldValues>({
```

Comment:
I believe this component was done in your other PR, right? In that case make the base branch for this PR to be the branch of your other PR

Reply:
That PR should be closed. We decided to implement everything from scratch in this PR